docs/guide/modeling.md

```diff
@@ -25,16 +25,19 @@ For example, an affected body might fly around as though it has life on its own,
 
 In order for Webots simulation results to be reproducible, the following conditions must be fulfilled:
 
-1. Each simulation must be restarted either by pushing the `Reload` button, or by using the `wb_supervisor_world_reload` function, or by restarting Webots.
+1. Each simulation must be executed with the same version of the Webots software, on the same machine.
+Running the same simulation on different machines or different Webots versions may result small numerical differences.
+However, it is possible to run fully reproducible results on different machines if these machines have the same operating system (same version, same updates), same graphics card and same graphics driver.
+It turns out that different OpenGL drivers or different graphics cards produce slightly different 3D rendering results.
+So, if your simulation doesn't rely on OpenGL rendering (no camera, lidar or range-finder), it should be reproducible on machines with a different graphics configuration.
+2. Each simulation must be restarted either by pushing the `Reload` button, or by using the `wb_supervisor_world_reload` function, or by restarting Webots.
 The random seeds used by Webots internally are reset for each simulation restarted with one of the above methods.
-2. The `synchronization` flag of every robot and supervisor must be TRUE.
+3. The `synchronization` flag of every robot and supervisor must be TRUE.
 Otherwise the number of physics steps per control step may vary with the current CPU load and hence the robot's behavior may also vary.
```

Comment:
```suggestion
Otherwise, the number of physics steps per control step may vary with the current CPU load and hence the robot's behavior may also vary.
```